### PR TITLE
Android TV: Implement new UI for Main Activity.

### PR DESCRIPTION
**Pull request by sigmabeta**
Pretty self explanatory stuff. Here's some shots:

Imgur

Imgur

Lots more to come: filling in the background with additional content, voice search accessible at the top of the sidebar, etc.

### CODE REVIEW

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java [hunk 1]
```diff
@@ -29,14 +37,22 @@
...
 
 	// So that MainActivity knows which view to invalidate before the return animation.
 	private int mPosition;
 
+	private boolean mDeviceHasTouchScreen;
+	private boolean mSystemUiVisible;
+	private boolean mMenuVisible;
+
+	private static Interpolator sDecelerator = new DecelerateInterpolator();
+	private static Interpolator sAccelerator = new AccelerateInterpolator();
```
Comment (lioncash) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java]: These aren't used elsewhere as far as I can tell

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/adapters/SettingsRowPresenter.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		holder.cardParent.setTitleText(resources.getString(settingsItem.getLabelId()));
+		holder.cardParent.setMainImage(resources.getDrawable(settingsItem.getIconId(), null));
+	}
+
+	public void onUnbindViewHolder(Presenter.ViewHolder viewHolder)
+	{
+		// no op
+	}
+
+
```
Comment (lioncash) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/adapters/SettingsRowPresenter.java]: unnecessary newlines

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/model/TvSettingsItem.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
+package org.dolphinemu.dolphinemu.model;
+
+
+public class TvSettingsItem
+{
+	private int mItemId;
+	private int mIconId;
+	private int mLabelId;
```
Comment (lioncash) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/model/TvSettingsItem.java]: These can be made final (and so can the class iirc)

**Comment by sigmabeta** [comment]
@dolphin-emu-bot rebuild

**Pull request merged**